Sorts/(new)mergesort.java

```diff
@@ -0,0 +1,99 @@
+package algorithm_homework;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.StringTokenizer;
+
+public class mergesort {
```

Comment:
In java class-names begins with upper case. Change the file name according to the class name!